IdentityCore/src/validation/MSIDAADAuthority.m

```diff
@@ -234,6 +234,16 @@ - (BOOL)supportsMAMScenarios
     return YES;
 }
 
+- (BOOL)checkTokenEndpointForRTRefresh:(NSURL *)tokenEndpoint
```

Comment:
Does it mean to check endpoint to refresh the refresh token?

Reply:
yep, checking the token endpoint for RT refresh scenario.